addons/web_editor/static/src/js/editor/odoo-editor/test/spec/insert.test.js

```diff
@@ -72,7 +72,7 @@ describe('insert HTML', () => {
                 stepFunction: async editor => {
                     await editor.execCommand('insert', parseHTML(editor.document, '<div><p>content</p></div>'));
                 },
-                contentAfter: '<div><p>content</p></div><p>[]<br></p>',
+                contentAfter: '\u200b<div><p>content</p></div><p>[]<br></p>',
```

Comment:
If you remember we discussed about a chromium bug which does not allow selection if container with `contenteditable false` is `firstChild` or `lastChild` of `contenteditable true` container. [here](https://discord.com/channels/1024949713759375421/1174681835876388874/1239517564304887829)

Do you want me to check for `banner` and `clipboard` explicitly rather than making it generic.